src/Twig/Extensions.php

```diff
@@ -116,9 +116,40 @@ public function getFilters()
                 'maintenanceModeColor',
                 [$this, 'maintenanceModeColor']
             ),
+            new \Twig\TwigFilter(
+                'orTemplateIfNotExists',
+                [$this, 'doesTwigFileExist']
+            ),
         );
     }
 
+    /**
+     * Does the template exist, or else return base template.
+     *
+     * @return string Filename of basepath, or default.
+     */
+    public function doesTwigFileExist(string $file, $default) : string
```

Comment:
Two parameters in the method's signature, but only 1 argument passed when this method is called?

Reply:
Right you are. Not sure why GCS didn't catch that.

Reply:
Oh, misread that, yes, so because this is a filter, the first argument is the value being filtered, and the second argument are the options passed in the function name.